contracts/MarginBase.sol

```diff
@@ -133,6 +133,15 @@ contract MarginBase is MinimalProxyable, OpsReady {
     /// @param required: amount of margin asset required
     error InsufficientFreeMargin(uint256 available, uint256 required);
 
+    /// @notice size cannot be zero when placing a limit/stop order
+    error SizeCannotBeZero();
```

Comment:
Might want to consider merging this SizeCannotBeZero with the AmountCannotBeZero -> ValueCannotBeZero. Then add it to the modifier I created that was merged into main

Reply:
Ended up adding an additional param to `ValueCannotBeZero` that way if there's multiple parameters to a function it's clear which one the error is referring to